### PR TITLE
Add TestAndTrace to api output

### PR DESCRIPTION
**Pull request by TomGoBravo**
## Background

See https://trello.com/c/eYwZ8roa/108-add-number-of-tracers-employed-to-our-api and 
https://covidactnow.slack.com/archives/C0110QQAPJA/p1589227207223300

### Please Check
- [x] Are the [JSON schemas](https://github.com/covid-projections/covid-data-model/tree/master/api/schemas) updated?
- [ ] Are the [api docs](https://github.com/covid-projections/covid-data-model/blob/master/api/README.V1.md) updated? Not really, docs seem to be in flux
- [x] Are tests passing?

### CODE REVIEW

File: api/can_api_definition.py [hunk 1]
```diff
@@ -79,6 +79,10 @@ class _Actuals(pydantic.BaseModel):
     )
     hospitalBeds: Optional[_ResourceUtilization] = pydantic.Field(...)
     ICUBeds: Optional[_ResourceUtilization] = pydantic.Field(...)
+    # contactTracers count is available for states, not counties.
+    contactTracers: Optional[int] = pydantic.Field(
```
Comment (ghop02) [inline api/can_api_definition.py:83]: You'll also have to add this into the place where actuals get set.  see [here](https://github.com/covid-projections/covid-data-model/blob/master/libs/functions/generate_api.py#L89)

File: libs/datasets/combined_datasets.py [hunk 2]
```diff
@@ -85,6 +86,7 @@
     CommonFields.ICU_TYPICAL_OCCUPANCY_RATE: [],
     CommonFields.POSITIVE_TESTS: [CDSDataset, CovidTrackingDataSource],
     CommonFields.NEGATIVE_TESTS: [CDSDataset, CovidTrackingDataSource],
+    CommonFields.CONTACT_TRACERS_COUNT: [TestAndTraceData],
```
Comment (ghop02) [inline libs/datasets/combined_datasets.py:89]: Can you add this to the latest values dataset above as well?

Comment (TomGoBravo) [reply libs/datasets/combined_datasets.py:90]: Done

File: api/can_api_definition.py (hunk 1 above)
Comment (TomGoBravo) [reply api/can_api_definition.py:83]: Done, thank You for the pointer.

File: libs/datasets/sources/test_and_trace.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        data[cls.Fields.COUNTRY] = "USA"
+        data[cls.Fields.AGGREGATE_LEVEL] = AggregationLevel.STATE.value
+        return data
+
+    @classmethod
+    def local(cls):
+        data_root = dataset_utils.LOCAL_PUBLIC_DATA_PATH
+        input_path = data_root / cls.DATA_PATH
+        data = pd.read_csv(
+            input_path, parse_dates=[cls.Fields.DATE], dtype={cls.Fields.FIPS: str}
```
Comment (goldblatt) [inline libs/datasets/sources/test_and_trace.py:47]: is this data per fips?

Comment (TomGoBravo) [reply libs/datasets/sources/test_and_trace.py:47]: It is state level only. The rows have fips and state set to the correct 2 character codes.

**Pull request merged**